fit/core-reference/src/test/java/org/apache/syncope/fit/core/UserIssuesITCase.java

```diff
@@ -1820,4 +1822,21 @@ void issueSYNCOPE1818() {
             jdbcTemplate.update("DELETE FROM TESTPULL WHERE USERNAME = 'rossini'");
         }
     }
+
+    @Test 
+    void issueSYNCOPE1830() {
```

Comment:
Please move this test method to `SearchITCase` and name it accordingly, e.g. `byMembershipAttribute()` or so